project/IgluSchemaPlugin.scala

```diff
@@ -0,0 +1,52 @@
+import sbt._
+import sbt.Keys._
+import sbt.io.IO
+import scala.sys.process._
+import scala.util.matching.Regex
+
+object IgluSchemaPlugin extends AutoPlugin {
```

Comment:
TODO: Move this plugin into [sbt-snowplow-release](https://github.com/snowplow-incubator/sbt-snowplow-release) and then import it here.